ui/app/AppLayouts/Wallet/popups/TransactionAddressMenu.qml

```diff
@@ -0,0 +1,371 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.13
+import QtQuick.Controls 2.14
+import QtQuick.Window 2.12
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+
+import shared.controls 1.0
+import utils 1.0
+import shared.stores 1.0
+
+import "../stores" as WalletStores
+
+StatusMenu {
+    id: root
+
+    property var sendModal
+    property var contactsStore
+
+    // TODO get those names from model
```

Comment:
Do you know if the TODO in this component is tracked somewhere in the associated tasks so that we don't lose track of them?

Reply:
I will search for any task. 
If nothing exists I will add something.